### DISCUSSION TITLE
Adding a border/overlay for Sinden lightgun usage

### DISCUSSION THREAD

**Discussion by StormedBubbles** [stella-emu/stella#863] [Ideas]
Hello,

I noticed that lightgun support was added back in 2020. It works great for me with a mouse! Thanks!

I have a question specific to a mouse device I have called a "Sinden Lightgun." When a thin white border is superimposed on the screen, the gun is able to detect the border via a built-in camera and position the mouse cursor where you aim. It works very well with various emulators. The gun uses its own software running in the background to process all of this, so if I could put such a border on, say, Sentinel in Stella, then it would just work because, as far as my computer is concerned, the gun is just a mouse.

Of note is that Sentinel already has a gray background surrounding the screen, which the gun can see somewhat. I get erratic cursor movement, which is a symptom of the border not being bright enough with respect to the rest of the game.

I guess the best way to phrase this would be: would it be possible to add a feature for custom overlays (.png) on games in Stella? If this already exists, then I am sorry for missing it! I looked through the various settings and didn't notice it.

**Comment by thrust26** [comment]
No such a feature does not exist yet. The request is a bit similar to [this](https://github.com/stella-emu/stella/issues/839), but I suppose the bezel would have to extend to top and bottom of the screen too.

**Reply by StormedBubbles** [reply]
Ah, "bezel" was a word on the tip of my tongue that I forgot to search for. Yes, that would be the same thing. If you replace the Atari side artwork with transparency and recolor that black border immediately around the game area to white, you essentially have a border that the Sinden gun could use. Being able to add any 1920x1080 .png (or whatever your screen size is) would allow users to resize the border as they see fit.

**Comment by thrust26** [comment]
Done with Stella 7.0. Testers welcome!
